src/Routers/FunctionsRouter.js

```diff
@@ -26,19 +26,19 @@ export class FunctionsRouter extends PromiseRouter {
       }
     }
   }
-  
+
   static handleCloudFunction(req) {
     var applicationId = req.config.applicationId;
     var theFunction = triggers.getFunction(req.params.functionName, applicationId);
     var theValidator = triggers.getValidator(req.params.functionName, applicationId);
     if (theFunction) {
-
       const params = Object.assign({}, req.body, req.query);
       var request = {
         params: params,
         master: req.auth && req.auth.isMaster,
         user: req.auth && req.auth.user,
-        installationId: req.info.installationId
+        installationId: req.info.installationId,
+        log: req.config.loggerController && req.config.loggerController.adapter
```

Comment:
You think we should expose the loggerController so we can perform additional tasks if needed instead of forwarding directly the 'unsafe' adapter. We try, as much as we can to avoid exposing the adapters directly.